### ISSUE TITLE
Add check for using the right name for groups of things

### ISSUE THREAD

**Issue by suchow** [amperser/proselint#132]
e.g., a group of cats is a "glaring".

http://www.oxforddictionaries.com/us/words/what-do-you-call-a-group-of

**Comment by j10sanders** [comment]
Do you really want these all implemented?  Most are humorous/antiquated terms that wouldn't be used seriously in modern prose.

**Comment by suchow** [comment]
Yes! It's true that some venery terms are so rare that they might come across as humor, but plenty are still in modern use and better than generic terms such as "bunch" — e.g., a swarm of bees, a herd of deer, a flock of sheep.

**Comment by j10sanders** [comment]
Ok!  

I could do this with a list structure of the various terms and their recommended replacements (like most other proselint modules), but I think this would be a good place to start using `pattern`
http://www.clips.ua.ac.be/pages/pattern-en#quantify

@suchow, do you agree?

**Comment by suchow** [comment]
Do you think [spaCy](https://spacy.io) could fill the same role? It's something I've been interested in integrating with Proselint for some time.

**Comment by j10sanders** [comment]
I've been messing around with spaCy for the last hour... it's very cool.  The main drawback for this project is that the package uses about 1.5 GB of disk space.

**Comment by j10sanders** [comment]
I'm going to do this the manual way

**Comment by j10sanders** [comment]
@suchow, do you have a moment to review my pull request?  Thanks!
